website/docs/design-patterns/partial-stack-configuration.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Partial Stack Configuration Atmos Design Pattern
+sidebar_position: 12
+sidebar_label: Partial Stack Configuration
+description: Partial Stack Configuration Atmos Design Pattern
+---
+
+# Partial Stack Configuration
+
+The **Partial Stack Configuration** Design Pattern describes the mechanism of splitting an Atmos top-level stack's configuration across many Atmos
+stack manifests to manage and modify them separately and independently.
+
+Each partial top-level stack manifest imports or configures a set of related Atmos components. Each Atmos component belongs to just one of the partial
+top-level stack manifests. The pattern helps to group all components by category or function and to make each partial stack manifest smaller and
+easier to manage.
+
+## Applicability
+
+Use the **Partial Stack Configuration** pattern when:
+
+- You have top-level stacks with complex configurations. Some parts of the configurations must be managed and modified independent of the other
+  parts, possibly by different people or teams
+
+- You need to group the components in a top-level stack by category or function
+
+- You want to keep the configuration easy to manage and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
+
+## Example
+
+In the following structure, we have many Terraform components (Terraform root modules) in the `components/terraform` folder.
+
+In the `stacks/catalog` folder, we define the defaults for each component using the [Component Catalog](/design-patterns/component-catalog) Atmos
+Design Pattern.
+
+In the `stacks/orgs/acme/plat/dev` folder, we split the `us-east-2` stack manifest into the following parts by category:
+
+- `us-east-2-load-balancers.yaml`
+- `us-east-2-data.yaml`
+- `us-east-2-dns.yaml`
+- `us-east-2-logs.yaml`
+- `us-east-2-notifications.yaml`
+- `us-east-2-firewalls.yaml`
+- `us-east-2-networking.yaml`
+- `us-east-2-eks.yaml`
```

Comment:
IMO, these should be in a `us-east-2/` folder and it's what we do in our refarch. See https://github.com/cloudposse-examples/infra-demo-eks/tree/main/stacks/orgs/acme/plat/prod/us-east-1